packages/components/src/globals/scss/_theme-tokens.scss

```diff
@@ -158,6 +158,12 @@
   /// @group accordion
   $accordion-arrow-margin: 0 $carbon--spacing-05 0 0 !default !global;
 
+  /// Computed accordion header height after the header has been built
+  /// @type Value
+  /// @access public
+  /// @group accordion
+  $accordion-heading-height: $carbon--spacing-07 !default !global;
```

Comment:
Did we confirm with design if these things are related at all? Would be interesting if we could use our spacing tokens in this way or not, mostly because I think some spacing tokens don't hit the pixel grid.

Reply:
After reading the guidelines, I think we can be more clear how spacing tokens should be used within a component, and layout tokens should be used outside components as you assemble pages. That's my understanding. I'll open a separate PR to confirm.

Reply:
While I was working on the technical audit I leaned towards using spacing tokens, but after the fact, I also wondered if it made more sense to use the mini-unit function instead?

```scss
$accordion-heading-height: miniUnits(4) !default !global;
```

Reply:
We discussed this in the morning and personally, I don't think spacing tokens are *intended* for this use (which is strictly speaking a height), but because it still influences the component, I can see an argument using it. I like the option of `miniUnits(4)`, more clear it's 32px at a glance.

Reply:
@shixiedesign is there anything design-related (or token related) that this decision would be based off of that we could parameterize? The decision in this case being the height of the accordion header

Reply:
I think we don't tokenize a component height because we expect line wrap during translations. This is why we center text in a component using `padding-top` and `padding-bottom`. When `padding-top` and `padding-bottom` are used to center an element, it's often random/odd values like 6px or 3px. It make sense to have a function to calculate it from a desired final height. But we should make sure the height is not a hard coded value to ensure translation & line wrap is not broken. FYI @mattrosno 

To answer your question, @joshblack we try to keep our component heights to multiples of eight (assuming no line wrap). We could potentially create parameters for heights, maybe just use data table's? Below are heights used in data table row, which are also common height values for a lot of our components (button, accordion, inputs, etc).

![image](https://user-images.githubusercontent.com/15144993/62079917-4436a500-b215-11e9-81a3-208eb767be5e.png)

Reply:
@shixiedesign I was moreso curious what factors would design use that influence the height of a panel, and maybe we could use that to determine what it should be for a given font face / font size / line-height pairing.

For example, the padding has been updated to derive from the type token's line-height and subtracting that from the accordion heading height.

Was curious if there was some kind of design decision we could embody in code that says if you're token is this font size / line height, then you're height should be ___.

Reply:
@shixiedesign `$accordion-heading-height` doesn't tokenize the `height` CSS property in this PR. It's used to set top and button paddings. The end result is no change for it's current styling; everything will still wrap as expected.

Reply:
I'll use `carbon--mini-units(4)` instead of `$carbon--spacing-07` and move this discussion to a separate PR after evaluating our current use of spacing tokens across all components.

Our components are very inconsistent on when we use mini unit functions, or spacing tokens, or layout tokens.

Reply:
@joshblack I see. I think there isn't 1-1 relationship between typography and height of a component. Height of a component is much more dependent on its function, so unfortunately it might not be easily computed. It's safe to say there are min height requirements. Eg. 14px type –> min height of a box containing that type should be 24px. 16px type –> min height should be 32px. but what we end up picking as the height of the component is usually based on things like how visible it should be? Eg. button should be more visible / bigger than accordion. What's usually around this component? Eg. button height should match input field if they line up. Does that answer your question?